packages.ini

```diff
@@ -135,6 +135,13 @@ apt_requires =
 brew_requires = wget
 custom_prebuild = prebuild/librdkafka v1.9.2
 python_versions = <3.11
+[confluent-kafka==2.1.1]
+apt_requires =
+    patch
+    wget
+brew_requires = wget
+custom_prebuild = prebuild/librdkafka v2.1.1
+python_versions = <3.11
```

Comment:
@asottile-sentry Do you know if this is all needed?

Reply:
the last line should not be -- but everything else is yes